bin/release

```diff
@@ -19,6 +19,8 @@ rake release
 
 docker build --rm -t codeclimate/codeclimate .
 docker push codeclimate/codeclimate
+docker tag codeclimate/codeclimate "codeclimate/codeclimate:$version"
+docker push "codeclimate/codeclimate:$version"
```

Comment:
Very good point: we actually had a debug tag of mine on docker hub already because of this! Fixed.